libs/feature/teaching/src/lib/lesson/forms/lesson-content.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { FormProvider, useForm } from "react-hook-form";
+import { LessonDescriptionForm } from "./lesson-content";
+
+// Define a wrapper component that initializes the form context
+function FormWrapper({
+	defaultValues,
+	children
+}: {
+	defaultValues: any;
+	children: React.ReactNode;
+}) {
+	const methods = useForm({ defaultValues });
+	return <FormProvider {...methods}>{children}</FormProvider>;
+}
+
+describe("LessonDescriptionForm", () => {
+	test('renders "aktivierungsfrage-element" when lessonType is "SELF_REGULATED"', () => {
```

Comment:
wir verwenden normalerweise `it` und dann im Stil `it should show "aktivierungsfrage" when lesosnType is SELF_REGULATED`

```suggestion
	it('should show "aktivierungsfrage" input when lesosnType is SELF_REGULATED'), () => {}
```

Reply:
bitte für alle tests anpassen